### PR TITLE
feat : Landing Page

### PR DESCRIPTION
**Pull request by mohammed-sas**
1. added hero image and featured categories
2. made the page responsive
3. csb : https://k02du.csb.app/
Screen Shot 2022-02-06 at 4 18 14 PM
e

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+.card-footer-basic .btn{
+    flex-grow: 1;
+    background: var(--primary);
+}
+.card-footer-basic .btn:hover{
+    background: var(--secondary);
+}
+.bg-black{
+    background: var(--black);
+}
```
Comment (starver20) [inline index.css:54]: This class can be added to body also, since there you are using same property

**Review by starver20 — APPROVED** [review]
Looks all good, just that class bg-black can be applied on body.

File: index.css (hunk 1 above)
Comment (mohammed-sas) [reply index.css:54]: since im importing my component library , adding bg-black in body tag will be overridden so i'd to add seperately

**Comment by gadgetfather** [comment]
Suggestion: you can remove that border gap from the bottom 
![image](https://user-images.githubusercontent.com/77927918/152677878-aec42822-8eca-472b-84f0-e8d007b97f17.png

**Review by gadgetfather — APPROVED** [review]

**Pull request merged**